101-bundle-json.md

```diff
@@ -442,16 +442,18 @@ Implementations MAY support user-defined additional actions as well. Such action
 
 ```json
 "actions": {
-    "status":{
+    "io.cnab.status":{
         "modifies": false
     },
-    "migrate":{
+    "io.cnab.migrate":{
         "modifies": false
     }
 }
 ```
 
-The above declares to actions: `status` and `migrate`. This means that the associated invocation images can handle requests for `status` and `migrate` in addition to `install`, `upgrade`, and `uninstall`.
+The action _name_ SHOULD be namespaced by the CNAB implementation which generated the bundle. 
+
+The above declares two actions: `io.cnab.status` and `io.cnab.migrate`. This means that the associated invocation images can handle requests for `status` and `migrate` in addition to `install`, `upgrade`, and `uninstall`.
```

Comment:
There needs to be a normative definition of namespacing. [Reverse domain names](https://en.wikipedia.org/wiki/Reverse_domain_name_notation) are reasonable, but the spec needs to be clear so that users don't choose varying namespacing schemes.

Reply:
I believe OCI specifies these SHOULD use reverse domains (https://github.com/opencontainers/image-spec/blob/master/annotations.md). But I think @mattfarina was in favor of staying less stringent and just suggesting "unambiguous dotted namespaces".

Reply:
That's true, but as namespacing is non mandatory (`SHOULD`), I don't if we can enforce a particular schema, other than just saying "Reverse domain names is preferred".  WDYT?

Reply:
I agree.

Reply:
I like the idea of reverse domain names being preferred. I can update the namespacing PR to reflect that.

Reply:
@mattfarina Okay. That would be good... or did I merge that one already?